README.md

```diff
@@ -10,9 +10,10 @@ Aleph Zero Blockchain python scripts for running TestNet and DevNet.
 - Make sure you have the aleph node binary inside a bin directory (needed for committee key generation step, `cp <...>/aleph-node/target/release/aleph-node bin/`)
 - Install packages needed for orchestrating experiments: GNU parallel, fabric, zip, unzip and Python 3 packages: fabric, boto3, ipython, tqdm, matplotlib, joblib.
 - Then, run `ipython -i shell.py`. This opens a shell with procedures orchestrating experiments.
-  The main procedure is `run_protocol(n_processes, regions, instance_type, volume_size, tag)` that runs `n_processes` spread
+  The main procedure is `setup_nodes(n_processes, regions, instance_type, volume_size, tag)` that prepares the `n_processes` spread
   uniformly across specified `regions` using EC2 machines of `instance_type`. E.g.
   `run_protocol(4, ['eu-west-1'], 't2.micro', 8, 'my_devnet')`.
```

Comment:
```suggestion
  `setup_nodes(4, ['eu-west-1'], 't2.micro', 8, 'my_devnet')`.
```

Reply:
Changed.

Reply:
my apologies for nitpicking, but as argument for `chain_type` is missing, all the further arguments will be misinterpreted, and the user will be confused with the error :(